src/sentry/static/sentry/app/views/settings/incidentRules/details.tsx

```diff
@@ -14,6 +14,7 @@ type RouteParams = {
 
 type Props = {
   organization: Organization;
+  onChangeTitle: ({ruleName: string}) => void;
```

Comment:
I think this type is wrong since it is called with a string: https://github.com/getsentry/sentry/pull/22210/files#diff-59a592c3b8b0fa2057fd8e5fe4fabdc6c50f98d10a64dd8530ef1f1e103948b0R41